modules/cli/src/main/scala/scala/cli/internal/CliLogger.scala

```diff
@@ -156,6 +156,15 @@ class CliLogger(
       def error(msg: String)    = logger.log(msg)
     }
 
+  val scalaNativeCliInternalLoggerOptions: List[String] = {
+    import scala.build.options.ScalaNativeOptions.Logger._
+
+    if (quiet) List(noDebug, noInfo, noWarn)
+    else if (verbosity >= 2) Nil
+    else if (verbosity == 1) List(noDebug)
+    else List(noDebug, noWarn)
```

Comment:
Let's only look at `verbosity`, that makes the code less ambiguous (not sure why we pass `quiet` to `CliLogger`):
```suggestion
    if (verbosity < 0) List(noDebug, noInfo, noWarn)
    else if (verbosity == 0) List(noDebug, noWarn)
    else if (verbosity == 1) List(noDebug)
    else Nil
```

Reply:
Actually, could we do it like this? I think we are going to want to keep the log level hierarchy intact on the sn-cli side. I think it's necessary to have the `info` log level as the default, but that only leaves us with the `debug` logs as more verbose.

``` scala
    if (verbosity < 0) List(noDebug, noInfo) // warn
    else if (verbosity == 0) List(noDebug) // info
    else Nil // debug
```
I imagine I could sneak some options to filter out the logs (like it is done in this PR), but I'd like to confirm that it is necessary to do so

Reply:
Sure!